### ISSUE TITLE
Updating to version v4.5-r1 causes error on installation

### ISSUE THREAD

**Issue by mattgig** [bfh/moodle-mod_verbalfeedback#47]
When you update to version v4.5-r1 on a Moodle 4.5.1 installation with MariaDB the update routine throws the following error:

```
mod_verbalfeedback
Error writing to database
More information about this error
Debug info: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'FROM md_verbalfeedback_i_category
WHERE md_verbalfeedback_i_category....' at line 1

UPDATE md_verbalfeedback_local_string SET instanceid = md_verbalfeedback_i_category.instanceid FROM md_verbalfeedback_i_category
WHERE md_verbalfeedback_i_category.id = md_verbalfeedback_local_string.foreignkey AND md_verbalfeedback_local_string.typeid = ?
[array (
0 => 2,
)]
Error code: dmlwriteexception×Dismiss this notification
Stack trace:
	• line 500 of /lib/dml/moodle_database.php: dml_write_exception thrown
	• line 293 of /lib/dml/moodle_read_slave_trait.php: call to moodle_database->query_end()
	• line 1258 of /lib/dml/mysqli_native_moodle_database.php: call to mysqli_native_moodle_database->query_end()
	• line 142 of /mod/verbalfeedback/db/upgrade.php: call to mysqli_native_moodle_database->execute()
	• line 119 of /mod/verbalfeedback/db/upgrade.php: call to add_instance_to_localized_string()
	• line 908 of /lib/upgradelib.php: call to xmldb_verbalfeedback_upgrade()
	• line 651 of /lib/upgradelib.php: call to upgrade_plugins_modules()
	• line 1937 of /lib/upgradelib.php: call to upgrade_plugins()
line 719 of /admin/index.php: call to upgrade_noncore()
```

Deinstalling the plugin and installing the new version works without a problem.

**Comment by srobotta** [comment]
Thank you for your comment and sorry about the installation issue.
I have tested it with Postgres only, however I will check to see if I can reproduce the error with a MySQL db.

**Comment by srobotta** [comment]
This has been fixed in the new release v4.5-r2
